features/order.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Taking an order
+  As a coffee shop employee
+  I want to be able to take orders
+  So I can fulfill the customer's needs
+
+  Scenario: Taking an order
+  Given I am on the homepage
```

Comment:
This is a perfect example of what we are looking for in regards to business logic, this is good.